charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -246,6 +246,14 @@ additionalVictoriaMetricsMap:
 
 # -- External VM read and write URLs
 externalVM:
+  # -- VMAuth config sections for external VM urls
+  vmauth:
+    read:
+      - src_paths:
+          - /.*
+        url_prefix:
+          - /
+    write: []
```

Comment:
I'd suggest enabling both routes by default in order to avoid confusion and add a comment clarifying a way to disable some of paths (similar to the same section in vmcluster config).